time_stamper_cv/include/common.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include "opencv2/opencv.hpp"
+
+class Common {
+ public:
+  static bool filter(double min, double max, double value) {
+    if (min > max) {
+      return false;
+    }
+    return value >= min && value <= max;
+  }
+};
+
+struct PointAngle {
+  cv::Point point;
+  double angle;
+};
+
+class PointVector : public std::vector<cv::Point> {
+ public:
+  PointVector() = default;
+
+  explicit PointVector(const std::vector<cv::Point>& keypoints) {
+    std::copy(keypoints.begin(), keypoints.end(), std::back_inserter(*this));
+  }
+
+  explicit PointVector(std::vector<cv::KeyPoint> keypoints) {
```

Comment:
param const

Reply:
This can't be const because otherwise the unary operator fails to compile